slashing-protection/src/integration-test/java/tech/pegasys/web3signer/slashingprotection/InterchangeV5ImporterIntegrationTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.slashingprotection;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import tech.pegasys.web3signer.slashingprotection.dao.SignedAttestationsDao;
+import tech.pegasys.web3signer.slashingprotection.dao.SignedBlocksDao;
+import tech.pegasys.web3signer.slashingprotection.dao.ValidatorsDao;
+import tech.pegasys.web3signer.slashingprotection.interchange.InterchangeV5Importer;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+
+import com.opentable.db.postgres.embedded.EmbeddedPostgres;
+import org.jdbi.v3.core.Jdbi;
+import org.junit.jupiter.api.Test;
+
+class InterchangeV5ImporterIntegrationTest extends InterchangeBaseIntegrationTest {
+
+  private final String basicJson =
```

Comment:
This is a bit hard to read, perhaps use the approach used in the InterchangeImportBadLogicalContentIntegrationTest and build it up programmatically?